variables.tf

```diff
@@ -19,8 +19,8 @@ variable "cluster_name" {
   type        = string
 }
 
-variable "cluster_security_group_id" {
-  description = "If provided, the EKS cluster will be attached to this security group. If not given, a security group will be created with necessary ingress/egress to work with the workers"
+variable "additional_security_group_id" {
```

Comment:
Why rename this variable ? I don't get it.

Reply:
Because they are 2 different things after 1.14 and you can not set the primary security group, only the additional security group id. The primary is created via eks automatically

Reply:
Because they are 2 different things after 1.14 and you can not set the primary security group, only the additional security group id. The primary is created via eks automatically

Reply:
https://www.terraform.io/docs/providers/aws/r/eks_cluster.html

vpc_config - Additional nested attributes:
cluster_security_group_id - The cluster security group that was created by Amazon EKS for the cluster.

You can not create this or even give an security group id for it, you can only get it from the output, so the additional_security_group_id is `security_group_ids` and what shows in the AWS console as additional security groups